src/web/server.ts

```diff
@@ -17,6 +17,7 @@ export async function startFastifyInstance(pluginsServer: PluginsServer): Promis
     status.info('👾', 'Starting web server…')
     const fastifyInstance = buildFastifyInstance()
     try {
+        // eslint-disable-next-line
```

Comment:
Could this only disable the offending rule (I'm not sure which one it is)?

Reply:
Fixed it in another way